README.md

```diff
@@ -126,6 +126,7 @@ This method produces Hjson text from a JavaScript value.
   - *space*: specifies the indentation of nested structures. If it is a number, it will specify the number of spaces to indent at each level. If it is a string (such as '\t' or '&nbsp;'), it contains the characters used to indent at each level.
   - *eol*: specifies the EOL sequence (default is set by Hjson.setEndOfLine())
   - *colors*: boolean, output ascii color codes
+  - *serializeDeterministically*: boolean, when serializing objects into hjson, order the keys based on their UTF-16 code units order. Default false.
```

Comment:
Is there a more-succinct name for this option? Like maybe `sortProps` or `sortKeys`?